packages/tooltip/src/Tooltip.tsx

```diff
@@ -332,7 +345,7 @@ function Tooltip({
                 className={cx(triangleStyle.notchStyle, notchVariants[variant])}
               />
             </div>
-            {children}
+            <Body className={childrenVariants[variant]}>{children}</Body>
```

Comment:
I'm not sure we're gonna want to always wrap children in a body component. We should consider scoping our solution to the ticket by simply applying the font size and color rules to a parent of children that already exists in the component.

Reply:
I'm fine with just applying those two rules, but I thought the point of the Body component was to standardize our typography usage, so this seemed like a great use-case for that.